g2p/tests/public/data/__init__.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
The code here was refactored from other parts of the `tests/` software. While not required for the mic tokenization problem, this refactoring work helped with the testing, so it made sense to include it in this PR.